packages/opentelemetry-cloud-monitoring-exporter/src/monitoring.ts

```diff
@@ -117,11 +117,21 @@ export class MetricExporter implements IMetricExporter {
       }
     }
 
+    let sendFailed: Error | undefined;
     for (const batchedTimeSeries of partitionList(
       timeSeries,
       MAX_BATCH_EXPORT_SIZE
     )) {
-      await this._sendTimeSeries(batchedTimeSeries);
+      try {
+        await this._sendTimeSeries(batchedTimeSeries);
+      } catch (err) {
+        err.message = `Send TimeSeries failed: ${err.message}`;
+        sendFailed = err;
+        this._logger.error(err.message);
```

Comment:
I think it might be more readable and less confusing behavior to create a new variable with the modified message to be logged rather than mutating the original error. I think modifying the caught error wouldn't do much unless it is re-thrown. Not a big deal either way, though.